recipes/spades/meta.yaml

```diff
@@ -5,29 +5,26 @@ package:
   version: {{ version }}
 
 source:
-  fn: SPAdes-{{ version }}-Darwin.tar.gz  # [osx]
-  url: http://cab.spbu.ru/files/release{{ version }}/SPAdes-{{ version }}-Darwin.tar.gz  # [osx]
-  sha256: 9e6accf2258c379436e92f37f920e4dc8fa4a2da9a9813215709a9723bf15d87  # [osx]
-  fn: SPAdes-{{ version }}.tar.gz  # [linux]
-  url: http://cab.spbu.ru/files/release{{ version }}/SPAdes-{{ version }}.tar.gz  # [linux]
-  sha256: 308aa3e6c5fb00221a311a8d32c5e8030990356ae03002351eac10abb66bad1f  # [linux]
+  fn: SPAdes-{{ version }}.tar.gz
+  url: http://cab.spbu.ru/files/release{{ version }}/SPAdes-{{ version }}.tar.gz
+  sha256: 308aa3e6c5fb00221a311a8d32c5e8030990356ae03002351eac10abb66bad1f
 
 build:
-  number: 1
-  string: py{{CONDA_PY}}_zlib{{CONDA_ZLIB}}_{{PKG_BUILDNUM}}  # [linux]
+  number: 2
+  string: py{{CONDA_PY}}_zlib{{CONDA_ZLIB}}_{{PKG_BUILDNUM}}
 
 requirements:
   build:
-    - gcc >=4.8.2  # [linux]
-    - cmake >=2.8.12  # [linux]
-    - zlib {{CONDA_ZLIB}}*  # [linux]
-    - bzip2 1.0.*  # [linux]
+    - gcc >=4.8.2
+    - cmake >=2.8.12
+    - zlib {{CONDA_ZLIB}}*
```

Comment:
pin to 1.2.8

Reply:
@epruesse: zlib is pinned correctly here per the [bioconda guidelines](https://bioconda.github.io/guidelines.html#c-c), via the value of the current environment variable defined in [env_matrix.yml](https://github.com/bioconda/bioconda-recipes/blob/master/scripts/env_matrix.yml). Explicitly pinning the version of zlib in the recipe would make mass rebuilds more difficult under a new version of zlib.